target/linux/armsr/armv7/config-6.1

```diff
@@ -21,13 +29,25 @@ CONFIG_ARM_LPAE=y
 CONFIG_ARM_PATCH_IDIV=y
 CONFIG_ARM_PATCH_PHYS_VIRT=y
 CONFIG_ARM_PSCI=y
+CONFIG_ARM_SMC_MBOX=y
+CONFIG_ARM_STM32_CPUIDLE=y
 CONFIG_ARM_THUMB=y
 CONFIG_ARM_UNWIND=y
 CONFIG_ARM_VIRT_EXT=y
 CONFIG_AUTO_ZRELADDR=y
 CONFIG_BINFMT_FLAT_ARGVP_ENVP_ON_STACK=y
 CONFIG_CACHE_L2X0=y
+CONFIG_CLKSRC_ARM_GLOBAL_TIMER_SCHED_CLOCK=y
```

Comment:
Why are you selecting this? I don't thing STM32 needs this or has this timer, but please explain I might be wrong.

Reply:
Removed